R/print.seroincidence.by.R

```diff
@@ -6,7 +6,7 @@
 #'
 #' @param x A list containing output of function [est.incidence.by()].
 #' @param ... Additional arguments affecting the summary produced.
-#'
+#' @returns Prints a list containing output of function [est.incidence.by()].
```

Comment:
```suggestion
#' @inherit print.seroincidence return
```

the `@inherit` tag here tells R to reuse the return value from `print.seroincidence()` for `print.seroincidence.by()` (details: https://cran.r-project.org/web/packages/roxygen2/vignettes/reuse.html)